### PR TITLE
Listener endpoint

### PR DESCRIPTION
**Pull request by annafossen**

### CODE REVIEW

File: message-collector/src/main/java/no/vegvesen/ixn/federation/messagecollector/MessageCollector.java [hunk 1]
```diff
@@ -1,7 +1,9 @@
 package no.vegvesen.ixn.federation.messagecollector;
 
 import no.vegvesen.ixn.federation.model.GracefulBackoffProperties;
+import no.vegvesen.ixn.federation.model.ListenerEndpoint;
 import no.vegvesen.ixn.federation.model.Neighbour;
```
Comment (kyrrek) [inline message-collector/src/main/java/no/vegvesen/ixn/federation/messagecollector/MessageCollector.java]: Remove unused imports. Neighbour is no longer here :-)

File: neighbour-service/src/main/java/no/vegvesen/ixn/federation/service/NeighbourService.java [hunk 2]
```diff
@@ -22,10 +25,7 @@
 import org.springframework.stereotype.Component;
 
 import java.time.LocalDateTime;
-import java.util.Collections;
-import java.util.List;
-import java.util.Optional;
-import java.util.Set;
+import java.util.*;
```
Comment (kyrrek) [inline neighbour-service/src/main/java/no/vegvesen/ixn/federation/service/NeighbourService.java:39]: Expand star imports

**Review by kyrrek — COMMENTED** [review]
Great work! Just do the small changes I commented on.

File: message-collector/src/main/java/no/vegvesen/ixn/federation/messagecollector/MessageCollector.java (hunk 1 above)
Comment (annafossen) [reply message-collector/src/main/java/no/vegvesen/ixn/federation/messagecollector/MessageCollector.java]: Done :)

File: neighbour-service/src/main/java/no/vegvesen/ixn/federation/service/NeighbourService.java (hunk 2 above)
Comment (annafossen) [reply neighbour-service/src/main/java/no/vegvesen/ixn/federation/service/NeighbourService.java:39]: Done :)

**Review by RogerMartinsen — APPROVED** [review]

**Pull request merged**